drivers/SmartThings/matter-switch/src/ElectricalEnergyMeasurement/server/attributes/CumulativeEnergyImported.lua

```diff
@@ -0,0 +1,68 @@
+local cluster_base = require "st.matter.cluster_base"
```

Comment:
Would we be able to remove the `CumulativeEnergyExported` and `PeriodicEnergyExported` embedded cluster defs now that we are just using the imported variation?

Reply:
We could. I figured why not leave them in, but it's just as well to take them out

Reply:
removed in [438b6252f08fba1d](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1830/commits/438b6252f08fba1dce8f2b398bef8a95ca1f05ce)